src/serac/numerics/equation_solver.cpp

```diff
@@ -1003,8 +1034,9 @@ void EquationSolver::defineInputFileSchema(axom::inlet::Container& container)
   iterative_container.addInt("max_iter", "Maximum iterations for the linear solve.").defaultValue(5000);
   iterative_container.addInt("print_level", "Linear print level.").defaultValue(0);
   iterative_container.addString("solver_type", "Solver type (gmres|minres|cg).").defaultValue("gmres");
-  iterative_container.addString("prec_type", "Preconditioner type (JacobiSmoother|L1JacobiSmoother|AMG|ILU).")
+  iterative_container.addString("prec_type", "Preconditioner type (JacobiSmoother|L1JacobiSmoother|AMG|ILU|Petsc).")
```

Comment:
```suggestion
  iterative_container.addString("prec_type", "Preconditioner type (JacobiSmoother|L1JacobiSmoother|AMG|ILU|PETSc).")
```

Reply:
personally, I think that for input files/CLI interface, it's not great to force users to remember the proper capitalization for their arguments. "PETSc" in particular is a pretty weird one that I expect will cause issues/annoyance. I think that all lowercase or all uppercase (or using case-independent conversion) would be a better choice.